paper/paper.tex

```diff
@@ -753,8 +761,8 @@ \subsubsection{Julia is fast}
 Julia is not generically slower than traditional high-performance programming
 languages like C, C++, and Fortran. A minor exception is due to
 Julia being built on top of LLVM, which does not perform the same optimizations as other
-C/C++/Fortran compilers. In particular, LLVM does
-not use fused multiply-add (FMA) instructions by default, while GCC and Intel
+C/C++/Fortran compilers. In particular, most LLVM frontends (including Julia) do
+not enable fused multiply-add (FMA) contraction by default, while GCC and Intel
 compilers do. Thus, one must opt-in to these optimizations explicitly.
 Relying on Julia's code manipulation techniques, this can be achieved with the macro
 \lstinline{@muladd}\footnote{\url{https://github.com/SciML/MuladdMacro.jl}}.
```

Comment:
I would also suggest the referring the reader to the docs for the `muladd` function (https://docs.julialang.org/en/v1/base/math/#Base.muladd), the `@muladd` macro just does some syntax rewriting to insert `muladd` function calls.

Reply:
Good point. I did that in 3499d50